indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -43,6 +43,20 @@ class Meta:
     show_if_id = fields.Integer(required=False, load_default=None, data_key='show_if_field_id')
     show_if_values = fields.List(fields.Raw(), required=False, data_key='show_if_field_values')
 
+    @validates('title')
+    @no_autoflush
+    def _check_unique_title_in_section(self, title, **kwargs):
+        from indico.modules.events.registration.models.form_fields import RegistrationFormItem
+        field = self.context['field']
+        query = (db.session.query(RegistrationFormItem)
+                 .filter(RegistrationFormItem.parent_id == field.parent.id,
+                         db.func.lower(RegistrationFormItem.title) == title.lower(),
+                         ~RegistrationFormItem.is_deleted))
```

Comment:
What do you think about disabled fields? Should these be excluded as well? If yes, it would of course also require a check when enabling a field.